examples/Azure_IoT_Central_ESP32/iot_configs.h

```diff
@@ -5,10 +5,54 @@
 #define IOT_CONFIG_WIFI_SSID              "SSID"
 #define IOT_CONFIG_WIFI_PASSWORD          "PWD"
 
+// Define if using device x509 certificates
+// #define IOT_CONFIG_USE_X509_CERT
+
+#ifdef IOT_CONFIG_USE_X509_CERT
+
+/*
+
+Device certificate should be of the form:
+
+"-----BEGIN CERTIFICATE-----\r\n" \
+"MIIBJDCBywIUfeHrebBVa2eZAbouBgACp9R3BncwCgYIKoZIzj0EAwIwETEPMA0G\r\n" \
+"A1UEAwwGRFBTIENBMB4XDTIyMDMyMjazMTAzN1oXDTIzMDMyMjIzMTAzN1owGTEX\r\n" \
+"MBUGA1UEAwwOY29udG9zby1kZXZpY2UwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNC\r\n" \
+.......
+"YmGzdaHTb6P1W+p+jmc+jJn1MAoGCXqGSM49BAMCA0gAMEUCIEnbEMsAdGFroMwl\r\n" \
+"vTfQahwsxN3xink9z1gtirrjQlqDAiEAyU+6TUJcG6d9JF+uJqsLFpsbbF3IzGAw\r\n" \
+"yC+koNRC0MU=\r\n" \
+"-----END CERTIFICATE-----"
+
+*/
+#define IOT_CONFIG_DEVICE_CERT "Device Certificate"
+
+/*
+
+Device certificate private key should be of the form:
```

Comment:
Try matching this description with the style used for documenting IOT_CONFIG_DEVICE_CERT, if you change the description according to the suggestions.